/**:
  ros__parameters:

    foot_print: [0.2, 0.4, 0.2, -0.4, 0.8, -0.5, 0.8, 0.5]

    leaf_size: 0.5

    use_debug: true

    x_min: 0.2

    x_max: 2.0

    y_min: -3.0

    y_max: 3.0

    z_min: -1.0

    z_max: 3.0
